wallet/db.c

```diff
@@ -200,6 +200,12 @@ static struct migration dbmigrations[] = {
 	 "  UNIQUE (payment_hash)"
 	 ");"),
      NULL},
+    {SQL("CREATE TABLE invoice_fallbacks ("
+	 "  scriptpubkey BLOB,"
+	 "  invoice_id BIGINT REFERENCES invoices(id) ON DELETE CASCADE,"
+	 "  PRIMARY KEY (scriptpubkey)"
+	 ");"),
+     NULL},
```

Comment:
This *MUST* go at the end of the dbmigrations array!!

Reply:
Yes, this was moved to the bottom in a later commit.